src/SignalR/clients/ts/signalr-protocol-msgpack/signalr-protocol-msgpack.npmproj

```diff
@@ -13,5 +13,9 @@
     <ProjectReference Include="..\signalr\signalr.npmproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <BuildOutputFiles Include="dist\browser\signalr-protocol-msgpack.js" />
```

Comment:
This doesn't include all of the build outputs, just more than what was here before.

Reply:
Curious: why not include all build output?